### PR TITLE
Also unescape commands starting with "! on Windows

### PR DESCRIPTION
**Pull request by GustavoLCR**
https://github.com/radare/radare2/issues/10613

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -547,7 +547,10 @@ class NewRegressions {
       if (process.platform === 'win32') {
         tests = tests.replace(/\/dev\/null/g, 'nul').replace(/\r\n/g, '\n').split('\n');
         for (let i = 0; i < tests.length; i++) {
-          if (!tests[i].slice(0, 1).localeCompare('!') || !tests[i].slice(0, 6).localeCompare('CMDS=!')) {
```
Comment (radare) [inline index.js:550]: this should be done in a separate function, looks a bit ugly to me

Comment (radare) [reply index.js:550]: but merging for now

**Comment by radare** [comment]
i think we will need another release of node-r2r and bump r2r

**Comment by GustavoLCR** [comment]
Yeah, sorry about that

**Pull request merged**